static/css/section/_management.scss

```diff
@@ -1,428 +1,274 @@
+
+  .row-ubuntu-advantage-promo {
+    @media only screen and (min-width : $breakpoint-large) {
+      overflow: hidden;
+
+      img.touch-border {
+        float: none;
+        max-width: none;
+        position: absolute;
+        right: 0;
+      }
+    }
+  }
+
+  .flush-right {
+    float: right;
+    margin-left: 20px;
+    margin-right: -40px;
+    }
+
+  .flush-left {
+    float: left;
+    margin-left: -40px;
+  }
 }
+// @section /management overview
+.management-home {
 
-body.management .row-landscape-trial-home div {
-	border-radius: 4px;
-	border: 1px solid #dfdcd9;
-	box-shadow: none;
-	margin-bottom: inherit;
-	margin-left: 0;
-	margin-right: 0;
 }
 
-/*	@section management-landscape-features
--------------------------------------------------------------- */
-body.management-landscape-features {
-  #inventory {
-  	margin-bottom: 30px;
+// @subsections /management/ubuntu-advantage, /management/working-with-landscape
+.management-ubuntu-advantage,
+.management-working-with-landscape {
+  .no-svg &,
+  .opera-mini & {
+    .row-hero {
+      background-image: url('#{$asset-server}2cb97b41-image-ubuntuadvantage.png');
+    }
   }
 }
 
-body.management-ubuntu-advantage,
-body.management-working-with-landscape {
-	.row-hero p.last {
-		margin-top: 40px;
-		background: none;
-	}
-	.row-hero .link-cta-canonical {
-		margin-bottom: 10px;
-	}
-	.ubuntu-advantage {
-  	margin-bottom: 0;
-  }
+// @subsections /management/compliance and /management/return-on-investment
+.management-compliance,
+.management-return-on-investment {
+
+  .no-margin-bottom {
+    margin-bottom: 20px;
 
-	.ubuntu-advantage img {
-    margin-left: -$gutter-width;
+    @media screen and (min-width: $breakpoint-medium) {
+      margin-bottom: 0;
+    }
   }
 
-	.row-purchase-advantage .box {
-		background: $light-grey url('#{$asset-server}3cc9f80f-pictogram-reduce-costs-115x115.png') 20px 20px no-repeat;
-		padding-bottom: 1em;
-	}
-	.row-purchase-advantage .promo-description {
-		padding-left: 135px;
-		border-right: 1px dotted $warm-grey;
-		padding-right: 1.1875em;
-		min-height: 9em;
-		margin-bottom: 0;
-	}
-	.row-purchase-advantage .promo-price {
-	}
-	.row-purchase-advantage .box div {
-		margin-bottom: 0;
-	}
-	.row-purchase-advantage .box p.last-item {
-		margin-top: 20px;
-		float: left;
-		font-style: italic;
-		font-size: .8125em;
-	}
-} /* end body.business-systems-management-ubuntu-advantage */
-html.no-svg,
-html.opera-mini {
-	body.management-ubuntu-advantage,
-	body.management-working-with-landscape {
-		.row-hero {
-			background-image: url('#{$asset-server}2cb97b41-image-ubuntuadvantage.png');
-		}
-	}
+  .no-svg &,
+  .opera-mini & {
+    .row-hero {
+      background: url('#{$asset-server}83697d44-pictogram-reduce-costs-300x300.png') no-repeat scroll 654px 50% $white;
+    }
+  }
 }
 
-body.management-ubuntu-advantage table {
-	table-layout: fixed;
+// @subsection /management/compliance
+.management-compliance {
+
+  img {
+    .flush-right {
```

Comment:
@pmahnke this structure implies markup like:

``` html
<img src="my-image">
  <span class="flush-right">Some content</span>
</img>
```

Which is invalid. `<img>` can't have children. Maybe what you meant was `img.flush-right {}`?